runner/context.ts

```diff
@@ -18,12 +18,17 @@ import {BrowserRunner} from './browserrunner';
 import * as config from './config';
 import {Plugin} from './plugin';
 
-type Handler = (o: {}, callback: (err: any) => void) => void;
+type Handler = (o: {}) => Promise<any>;
```

Comment:
what's the point of the empty anonymous interface vs `any`, or a named interface?